src/ScriptBuilder/CSharpCounterWriter.cs

```diff
@@ -6,7 +6,7 @@
 
     static class CSharpCounterWriter
     {
-        public static void WriteCode(string scriptPath, IEnumerable<TimerDefinition> timers, IEnumerable<MeterDefinition> meters, Dictionary<string, string> legacyInstanceNameMap)
+        public static void WriteCode(string scriptPath, IEnumerable<DurationDefinition> timers, IEnumerable<SignalDefinition> meters, Dictionary<string, string> legacyInstanceNameMap)
```

Comment:
I'm pretty sure this is not the type we want to use. That being said I know that @dvdstelt and @kbaley had a plan of fixing that with backwards compatibility in mind. I would like to hear their opinion on that.